Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -284,6 +284,11 @@ void VideoBackendBase::ActivateBackend(const std::string& name)
 
 void VideoBackendBase::PopulateBackendInfo(const WindowSystemInfo& wsi)
 {
+  // If the core is running, the backend info will have been populated already. If we did it here,
+  // the UI thread could race with the GPU thread.
+  if (Core::IsRunningOrStarting(Core::System::GetInstance()))
```

Comment:
Is it correct that you want to repopulate the backend info if the core is currently shutting down? I imagine this would have similar problems to doing it while the core is running.

Reply:
Good point, https://github.com/dolphin-emu/dolphin/pull/13078.